tests/test.py

```diff
@@ -23,30 +23,45 @@ def test_metadata():
     assert r.status_code == 200
 
     metadata = r.json()
-    assert metadata['id'] == "mask_rcnn_nucleus_detection-keras-model"
-    assert metadata['name'] == "mask_rcnn_nucleus_detection Keras Model"
-    assert metadata['description'] == "mask_rcnn_nucleus_detection Keras model trained on 2018 Data Science Bowl"
+    assert metadata['id'] == "max-nucleus-segmenter"
+    assert metadata['name'] == "MAX Nucleus Segmenter"
+    assert metadata['description'] == "Nucleus image segmentation model trained with Keras on 2018 Data Science Bowl dataset"
     assert metadata['license'] == 'Apache License 2.0'
 
 
 def test_predict():
 
     model_endpoint = 'http://localhost:5000/model/predict'
 
-    # Test by the image with multiple nucleis
-    img1_path = 'assets/example.png'
+    # Test by the image with multiple nuclei
+    img_png = 'assets/example.png'
+    img_jpg = 'assets/example.jpg'
```

Comment:
We can move the `jpg` and `tiff` images to the `tests` folder as they are only used for testing. We don't need to include 3 of the same images in `assets`